update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+/*
+This adds corresponding sia405 values for interlis import/export.
+*/
+
+-- Add columns
+
+ALTER TABLE qwat_vl.value_list_base ADD COLUMN sia405code TEXT;
+
+
+-- Populate columns
+
+-- TODO : Bedding material mapping complete [X] to check [ ]
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'aucun';
+UPDATE qwat_vl.bedding SET sia405code = 'Beton' WHERE value_fr = 'béton';
+UPDATE qwat_vl.bedding SET sia405code = 'Fliess' WHERE value_fr = 'géotextile';
+UPDATE qwat_vl.bedding SET sia405code = 'Betonkies' WHERE value_fr = 'gravier';
+UPDATE qwat_vl.bedding SET sia405code = 'Sand' WHERE value_fr = 'sable';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'terre';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cistern SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cover_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- Hydrant material : mapping complete [X] to check [ ]
+UPDATE qwat_vl.hydrant_material SET sia405code = 'Metall' WHERE short_fr LIKE 'F %';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_inf SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_sup SET sia405code = 'unbekannt' WHERE ???;
```

Comment:
I think all hydrant_model_sup are Hydrant.Art = Unterflurhydrant. We would need to fill in the subvalues in qwat for a detailled matching (einarmig, zweiarmig, mitEntleerung, ohne. Also we should take the Information about the Hersteller out of these values (e.g. VonRoll). My question is, whether we invest in the matching here or add addtional columns to qwat_vl.hydrant_model_sup for Art and Hersteller.

Reply:
Hydrant.Hersteller is already stored separately as hydrant_provider